### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Quinn-Donnelly**
Seeks to resolve (#1) 
Implements the design of the old homepage along with the added description.

- The search bar will route the user the the search page
- This method that it send the search conforms to the new search method layed out in (#21)

### CODE REVIEW

**Review by Jubstron — APPROVED** [review]
🤜 ... 👍

File: app/containers/HomePage/index.js [hunk 1]
```diff
@@ -1,28 +1,96 @@
...
+        <Helmet>
+          <title>Homepage</title>
+          <meta name="description" content="Description of Homepage" />
+        </Helmet>
+
+        <CenterImage>
+          <Image src={logo} responsive />
+        </CenterImage>
+        <Col md={8} mdOffset={2}>
+          <Description name="ieatass" innerStyle={{ className: 'text-center' }} text={{ short: 'Justin will write this later', long: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque totam natus exercitationem illo, voluptatibus, ullam molestias reiciendis deserunt, optio officia quam vel. Voluptates quidem nihil dignissimos libero, quaerat dicta! Quia, aperiam, doloremque. Dolorum odio incidunt, rem, doloremque amet delectus beatae nisi rerum eligendi. Neque non sequi optio numquam atque ipsa rerum accusamus magnam perspiciatis, ratione, officiis, eius? Adipisci temporibus ab optio id numquam, deleniti aspernatur modi nisi libero! Laudantium excepturi nesciunt fugit ab facilis porro cupiditate? Molestiae distinctio expedita veritatis illum aliquam saepe doloribus modi natus quae. Sunt, quis. Dolore consequuntur aut autem quis, iure reiciendis eligendi iste molestias incidunt!' }} />
```
Comment (bsonge) [inline app/containers/HomePage/index.js]: lol

**Review by bsonge — APPROVED** [review]
10 :cucumber: out of :cake:

File: app/containers/HomePage/index.js (hunk 1 above)
Comment (wrejcodes) [reply app/containers/HomePage/index.js]: name is still ieatass btw. Probably doesn't matter but we might not want that in the repo lolololol

**Review by wrejcodes — APPROVED** [review]
Overall it looks good except for that one potential change

**Pull request merged**